chain/chain/src/store.rs

```diff
@@ -875,6 +876,13 @@ impl ChainStoreAccess for ChainStore {
         option_to_not_found(self.store.get_ser(DBCol::BlockMisc, FINAL_HEAD_KEY), "FINAL HEAD")
     }
 
+    fn flat_state_head(&self) -> Result<CryptoHash, Error> {
+        option_to_not_found(
+            self.store.get_ser(DBCol::BlockMisc, FLAT_STATE_HEAD_KEY),
+            "FLAT STATE HEAD",
```

Comment:
Why do you need to store flat state head on disk? I think it's enough to have that only in memory.

Reply:
If node crashes, we need some key on disk to restore head.